appscale/tools/remote_helper.py

```diff
@@ -788,8 +810,8 @@ def terminate_cloud_infrastructure(cls, keyname, is_verbose):
       is_verbose: A bool that indicates if we should print the commands executed
         to stdout.
     """
-    AppScaleLogger.log("About to terminate instances spawned with keyname {0}".
-                       format(keyname))
+    AppScaleLogger.log("About to terminate deployment and instances with keyname {0}."
+      .format(keyname) + " Press Ctrl-C to stop.")
```

Comment:
Change to:
`"About to terminate deployment and instances with keyname {0}. Press Ctrl-C to stop.".format(keyname)`

No `+`ing for any kind is allowed. :p